src/api/operator/operatorApi.ts

```diff
@@ -222,18 +228,24 @@ export async function getOrders(params: GetOrdersParams): Promise<RawOrder[]> {
     response = await _get(networkId, queryString)
   } catch (e) {
     const msg = `Failed to fetch ${queryString}`
-    console.error(msg, e.message)
+    console.error(msg, e)
     throw new Error(msg)
   }
 
   if (!response.ok) {
-    throw new Error(`Request failed: [${response.status}] ${response.body}`)
+    // Just in case response.text() fails
+    const responseText = await response.text().catch((e) => {
```

Comment:
same as above, uncaught as it's not in a try/catch and the inlined `catch` is rethrowing

Reply:
It will read as uncaught in console as it's falling through the inline catch when it's rethrown in line 242. Solution is add it in try block or just leave as is. for me it's fine